app/css/index.scss

```
@@ -283,6 +283,14 @@
     float: right;
     font-size: 0.8em;
   }
+  .search-result-path {    
```

There are trailing whitespaces here:
```suggestion
  .search-result-path {
```